doc/source/fmpz_poly.rst

```diff
@@ -2085,6 +2085,16 @@ Derivative
 
     Sets ``res`` to the derivative of ``poly``.
 
+.. function:: void _fmpz_poly_nth_derivative(fmpz * rpoly, const fmpz * poly, ulong n, slong len)
+
+    Sets ``(rpoly, len - n)`` to the nth derivative of ``(poly, len)``.  
+    Also handles the cases where ``len <= n`` correctly. 
+    Supports aliasing of ``rpoly`` and ``poly``.
```

Comment:
Same here.

And by the way, are they supposed to be visible to the user? If its only purpose is to be a helper function to `fmpz_poly_nth_derivative`, I believe it should not be included (could make the documentation harder to read). Perhaps someone can correct me here if I'm wrong.

Reply:
Yes it's just a helper function for ```fmpz_poly_nth_derivative```, I added it to the doc for completeness because ```_fmpz_poly_derivative``` was already in there.